### PR TITLE
Login page update

### PR DESCRIPTION
**Pull request by MohamedCheour98**
I traced back the login button to be able to locate the problem and understand why the user wasn't prompted an error message when the credentials were not validated.

### CODE REVIEW

File: bbreadfront/src/SignUpForm.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
     job: ""
   });
+  
   const [show, setShow] = React.useState(false);
   const [redirect, setRedirect] = React.useState(false);
   const [userData, setUserData] = useState({});
+  
   async function submitForm() {
     let signedIn = await props.handleSubmit(person);
-    console.log(signedIn)
```
Comment (hannahtobiason) [inline bbreadfront/src/SignUpForm.js:13]: Thank for the refactoring of console logs and making sure the style is consistent. Looks good!

**Review by hannahtobiason — APPROVED** [review]
Approve! thanks for the fix

File: bbreadfront/src/LoginForm.js [hunk 2]
```diff
@@ -18,12 +18,12 @@ function LoginForm(props) {
   /*const message = this.state.displayErrorMessage ? null : <div> nice</div>; */
   // strange array prototype, making us use loggedIn[0], Joseph?
   async function submitForm() {
+    
     let loggedIn = await props.handleSubmit(person);
-    console.log(loggedIn)
-    if (Object.keys(loggedIn[0]).length != 0) {
+    
+    if (Object.keys(loggedIn).length != 0) {
```
Comment (hannahtobiason) [inline bbreadfront/src/LoginForm.js:24]: Tested this change locally, good to go!

**Review by hannahtobiason — APPROVED** [review]

**Comment by Eriken79** [comment]
I'm glad that bug got fixed. It's also great that all the build checks are passing. I'm not sure why I still can't view our app on www.breakbread1.herokuapp.com, but if it works for everyone else, that's just a me problem! Thanks for the refactoring as well that's super helpful for future additions to those files.

File: bbreadfront/src/LoginForm.js (hunk 2 above)
Comment (MohamedCheour98) [reply bbreadfront/src/LoginForm.js:24]: Awesome!

File: bbreadfront/src/SignUpForm.js (hunk 1 above)
Comment (MohamedCheour98) [reply bbreadfront/src/SignUpForm.js:13]: Yes, of course. It just looks better now and that way we don't waste time looking for previous console.log that we forgot to delete.

Comment (josephos416) [reply bbreadfront/src/SignUpForm.js:13]: Looks good to me. And yeah the console logs look good

**Pull request merged**